src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,26 @@
+.catalog {
+  padding: 50px 40px;
+
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+}
+
+@include on-mobile {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@include on-tabile {
```

Comment:
There seems to be a typo in the mixin name `on-tabile`. It should likely be `on-tablet` or another correct name as per your project's naming conventions.